pyramid_restful/api.py

```diff
@@ -187,6 +188,8 @@ def process(self, request):
                     if permit and not request.has_permission(permit):
                         raise HTTPForbidden()
                     else:
+                        if action:
```

Comment:
we should have this be a optional thing, if it doesn't find an action, then it falls back to the default action.  Reason being, "action" could also just be data to an endpoint, you could have a model that looks like:

```
class Object(Table):
     id = Id()
     action = String()
```

And doing a `POST /api/v1/objects?action=test` shouldn't fail if the user didn't use this feature.